actors/evm/src/interpreter/precompiles/fvm.rs

```diff
@@ -21,45 +21,39 @@ pub(super) fn get_actor_type<RT: Runtime>(
     input: &[u8],
     _: PrecompileContext,
 ) -> PrecompileResult {
-    const LAST_SYSTEM_ACTOR_ID: u64 = 32;
-
+    // should never panic
```

Comment:
Please always state why.

Reply:
well, it would panic, so now it is fixed and shouldn't panic.